dotnet/src/InternalUtilities/src/Schema/JsonSchemaMapper.ReflectionHelpers.cs

```diff
@@ -177,16 +177,16 @@ private static FieldInfo GetPrivateFieldWithPotentiallyTrimmedMetadata(this Type
     {
         public ParameterLookupKey(string name, Type type)
         {
-            Name = name;
-            Type = type;
+            this.Name = name;
+            this.Type = type;
```

Comment:
I was purposefully avoiding making changes to these files for style, so that we can minimize the diffs as we keep it in sync with @eiriktsarpalis's "official" version until it's part of STJ. I'd added an editorconfig that should have suppressed the warnings on these files; did that not work?

Reply:
My IDE was complaining about them, I can revert.